CHANGELOG.md

```diff
@@ -39,6 +39,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - semantics for CreateContextStateWithAssociationAndValidators manipulation
 - semantics for SetDeviceOperatingMode manipulation
 - message PartialIdentification to message PartialInstanceIdentifier
+- semantics for SetMetricStatus manipulation
```

Comment:
After the convention for Changelogs the latest entry comes first. So I would expect your change at the beginning of the Changed section.

Reply:
done